pkg/subctl/cmd/root.go

```diff
@@ -43,9 +43,10 @@ import (
 )
 
 var (
-	kubeConfig  string
-	kubeContext string
-	rootCmd     = &cobra.Command{
+	kubeConfig   string
+	kubeContext  string
+	kubeContexts []string
```

Comment:
I think it would be simpler to share a single slice for both of these (changing the type of `--kubecontext`, but it won’t change anything for users).

Reply:
> I think it would be simpler to share a single slice for both of these (changing the type of `--kubecontext`, but it won’t change anything for users).

Changing `kubecontext` type would require change in every single caller. That is why I wanted to know why we even need to pass it as an arg when it is already a global var in `cmd`